_clients/data-prepper/data-prepper-reference.md

```diff
@@ -37,22 +37,23 @@ Sources define where your data comes from.
+Option | Required      | Type    | Description
+:--- |:--------------|:--------| :---
+port | No            | Integer | The port OTel trace source is running on. Default is `21890`.
+request_timeout | No            | Integer | The request timeout in milliseconds. Default is `10_000`.
+health_check_service | No            | Boolean | Enables a gRPC health check service under `grpc.health.v1/Health/Check`. Default is `false`.
+proto_reflection_service | No            | Boolean | Enables a reflection service for Protobuf services (see [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) and [gRPC Server Reflection Tutorial](https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md) docs). Default is `false`.
+unframed_requests | No            | Boolean | Enable requests not framed using the gRPC wire protocol.
+thread_count | No            | Integer | The number of threads to keep in the ScheduledThreadPool. Default is `200`.
+max_connection_count | No            | Integer | The maximum allowed number of open connections. Default is `500`.
+ssl | No            | Boolean | Enables connections to the OTel source port over TLS/SSL. Defaults to `true`.
+sslKeyCertChainFile | Conditionally | String  | File-system path or AWS S3 path to the security certificate (e.g. `"config/demo-data-prepper.crt"` or `"s3://my-secrets-bucket/demo-data-prepper.crt"`). Required if ssl is set to `true`.
+sslKeyFile | Conditionally | String  | File-system path or AWS S3 path to the security key (e.g. `"config/demo-data-prepper.key"` or `"s3://my-secrets-bucket/demo-data-prepper.key"`). Required if ssl is set to `true`.
+useAcmCertForSSL | No            | Boolean | Whether to enable TLS/SSL using certificate and private key from AWS Certificate Manager (ACM). Default is `false`.
+acmCertificateArn | Conditionally | String  | Represents the ACM certificate ARN. ACM certificate take preference over S3 or local file system certificate. Required if `useAcmCertForSSL` is set to `true`.
+awsRegion | Conditionally | String  | Represents the AWS region to use ACM or S3. Required if `useAcmCertForSSL` is set to `true` or `sslKeyCertChainFile` and `sslKeyFile` are AWS S3 paths.
+authentication | No            | Object  | An authentication configuration. By default, this runs an unauthenticated server. This uses pluggable authentication for HTTPS. To use basic authentication, define the `http_basic` plugin with a `username` and `password`. To provide customer authentication use or create a plugin which implements: [GrpcAuthenticationProvider](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/armeria-common/src/main/java/com/amazon/dataprepper/armeria/authentication/GrpcAuthenticationProvider.java).
```

Comment:
By "this runs an authenticated server", do you mean Data Prepper runs on an authenticated server? And enabling `authentication` will use pluggable authentication for HTTPS?

Reply:
The authentication mechanisms is independent of HTTPS. The HTTPS is controlled by parameters such as `acmCertificateArn`.

Yes, the default pipeline will create an unauthenticated server, which means that any request is trusted.

Reply:
Can we say that explicitly? Something like, "The default pipeline creates an unauthenticated server. To use basic authentication, define the ...".

Reply:
Changed wording to By default, this creates an unauthenticated server for the pipeline